tests/error.js

```diff
@@ -0,0 +1,28 @@
+/*
+ * error.js
+ */
+
+
+const gi = require('../lib/')
+const GLib = gi.require('GLib', '2.0')
+const Gst = gi.require('Gst', '1.0')
+const common = require('./__common__.js')
+
+Gst.init()
+
+const ERROR_MESSAGE = 'ERROR_MESSAGE'
+const ERROR_CODE = 42
+const DEBUG_STRING = 'debug string'
+
+common.describe('GError', () => {
+  common.it('should be returned', () => {
+    common.assert(GLib.Error !== undefined)
+    const err0 = GLib.Error.newLiteral(404, ERROR_CODE, ERROR_MESSAGE)
```

Comment:
Error domains are `GQuark`s and need to be registered as such. You shouldn't allow passing random integers in here or otherwise code will likely misbehave or crash. For example you should get back a proper string when calling `g_quark_to_string()` on it.

Reply:
You are right. The testing is not done properly. It should be fixed through #227.